### PR TITLE
Move more release instructions to the template

### PR DESCRIPTION
**Pull request by raboof**
Sparked by discussion in #17432

### CODE REVIEW

**Review by johanandren — APPROVED** [review]

File: RELEASING.md [hunk 1]
```diff
@@ -53,20 +53,14 @@ your `~/.sbt/1.0/private-credentials.sbt`.
...
-### Release script instructions
-
-Make sure you have completed the setup in `project/scripts/release`.
-
 ## Snapshot releases
 
-Nightly snapshot releases are created from master and published to
-https://repo.akka.io/snapshots by https://jenkins.akka.io:8498/job/akka-publish-nightly/
+Snapshot releases are created from master and published to
+https://oss.sonatype.org/content/repositories/snapshots/com/typesafe/akka/
```
Comment (patriknw) [inline RELEASING.md:59]: could you also update the doc page about snapshots https://doc.akka.io/docs/akka/current/project/links.html#snapshots-repository

File: RELEASING.md [hunk 2]
```diff
@@ -53,20 +53,14 @@ your `~/.sbt/1.0/private-credentials.sbt`.
...
 
-To create snapshot versions manually, use `sbt clean publish`.
-The release artifacts are created in `akka-*/target/repository` and can be
-copied over to a maven server. If you have access, the Jenkins job at
-https://jenkins.akka.io:8498/job/akka-publish-wip/ can be used to publish
-a snapshot to https://repo.akka.io/snapshots from any branch.
+To create snapshot versions manually, use `sbt clean publishLocal`.
+If you have access, you can use `publishSigned` to publish them to
```
Comment (patriknw) [inline RELEASING.md:62]: `+publishSigned` ?

File: scripts/release-train-issue-template.md [hunk 3]
```diff
@@ -3,15 +3,18 @@ Release Akka $VERSION$
 ### Before the release
 
 - [ ] Make sure all important / big PRs have been merged by now
-- [ ] Create a news item draft PR on [akka.github.com](https://github.com/akka/akka.github.com), using the milestone
-- [ ] Make sure to update `versions.json` in it
+- [ ] Create a news item draft PR on [akka.io](https://github.com/akka/akka.io), using the milestone and `scripts/authors.scala v2.3.5 v2.3.6`
```
Comment (octonato) [inline scripts/release-train-issue-template.md:6]:
```suggestion
- [ ] Create a news item draft PR on [akka.io](https://github.com/akka/akka.io), using the milestone and `scripts/authors.scala v2.6.14 v2.6.15`
```

2.3 is such an old one :-)

**Review by octonato — APPROVED** [review]
LGTM

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Pull request merged**